doctrinal-works-v3/original/38-chapter-38.adoc

```diff
@@ -44,24 +44,40 @@ This is enough to let you understand my will, if you will be so wise as to know
 [.blurb]
 === An answer to the speech or declaration of the great Turk, etc. By G. F.
 
+[.embedded-content-document.letter]
+--
```

Comment:
Hmmm... You did exactly right in embedding this, but I sort of feel like because of the strange nature of this document, that it feels more correct _not_ embedded, like the previous one that you did not embed. Would you mind quick un-embedding this while you're doing the other correction too?  thanks!